plugins/modules/database/influxdb/influxdb_user.py

```diff
@@ -219,7 +224,23 @@ def main():
     grants = module.params['grants']
     influxdb = influx.InfluxDb(module)
     client = influxdb.connect_to_influxdb()
-    user = find_user(module, client, user_name)
+
+    user = None
+    try:
+        user = find_user(module, client, user_name)
+    except influx.exceptions.InfluxDBClientError as e:
+        if e.code == 403:
+            reason = None
+            try:
+                msg = json.loads(e.content)
+                reason = msg["error"]
+            except (json.decoder.JSONDecodeError, KeyError):
```

Comment:
If I run
```.py
import json
json.decoder.JSONDecodeError
```
in Python 2.7, I get `AttributeError: 'module' object has no attribute 'JSONDecodeError'`.

I guess it's better to just do a broad `except Exception:` here.

Reply:
Sorry, I did not test it with Python 2 😅 

Instead of catching `Exception` I have opted to handle `KeyError, ValueError`. `json.loads` returns a `ValueError` on Python 2. On Python 3 `json.decoder.JSONDecodeError` is a subclass of `ValueError`.